# LogMessage Message

std_msgs/Header header

std_msgs/String logMessage



# logMessage @18 :Text;
